config/software/openssl-customization.rb

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright:: Copyright (c) 2014 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This software makes sure that SSL_CERT_FILE environment variable is pointed
+# to the bundled CA certificates that ship with omnibus. With this, Chef
+# tools can be used with https URLs out of the box.
+name "openssl-customization"
+
+source path: "#{project.files_path}/#{name}"
```

Comment:
So I'm a little worried about this. It's not completely obvious that any project using this software def will have to cargo cult files from omnibus-chef. I wonder if we should make `openssl-customization` a dependency of the Chef project only. I don't think the server projects which will be leveraging the `chef` software definition need the openssl customization bits (which are Windows only).

Thoughts @jdmundrawala @thommay @btm?